Dockerfile

```diff
@@ -0,0 +1,132 @@
+# -------------------------------------------------------------------------------------------------
+# BUILDER IMAGE
+# -------------------------------------------------------------------------------------------------
+FROM openjdk:8-jdk-alpine as builder
+ARG VERSION="latest"
+
+###
+### Dependencies
+###
+RUN apk add --no-cache \
+	curl \
+	git
+
+###
+### Download
+###
+RUN set -eux \
+	&& if [ "${VERSION}" = "latest" ]; then \
+			DATA="$( \
+				curl -sS https://github.com/linkedin/cruise-control/releases \
+				| tac \
```

Comment:
Are both needed?

Reply:
Yes, otherwise it will be reversed: [tac](https://linux.die.net/man/1/tac)